### DISCUSSION TITLE
Disable selection of future dates (For DOB Input)

### DISCUSSION THREAD

**Discussion by merlin9900** [gpbl/react-day-picker#2583] [General]
I cant figure out what would be the best and optimized way to disable future dates, in NextJS as using dates is very complicated in it.

**Comment by sos0** [comment]
Have you checked this link by any chance? https://daypicker.dev/docs/selection-modes#disabling-dates-in-the-past

**Comment by merlin9900** [comment]
Thanks man!!!
